src/uiTest/resources/project.rhino.js

```diff
@@ -352,11 +352,19 @@ function Project(underlyingProject) {
     invokeActionAndWait('GitMachete.FastForwardMergeCurrentBranchToParentAction', ACTION_PLACE_TOOLBAR, {});
   };
 
-  this.mergeParentIntoSelectedBranchAction = function (branchName) {
+  this.syncSelectedToParentByRebaseAction = function (branchName) {
+    invokeActionAsync('GitMachete.RebaseSelectedBranchOntoParentAction', ACTION_PLACE_CONTEXT_MENU, { SELECTED_BRANCH_NAME: branchName });
```

Comment:
Huh btw, shouldn't the action classes be renamed for consistency?

Reply:
I think so too

Reply:
Pls issue

Reply:
pr instead: https://github.com/VirtusLab/git-machete-intellij-plugin/pull/886